doc/api/net.md

```diff
@@ -625,15 +625,16 @@ initialDelay will leave the value unchanged from the default
 
 Returns `socket`.
 
-### socket.setNoDelay([noDelay])
+### socket.setNoDelay([enable])
```

Comment:
it is too ambiguous otherwise and parameter name in code is `enable`: https://github.com/nodejs/node/blob/master/lib/net.js#L337